src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -7009,6 +7013,112 @@ HRESULT ProfToEEInterfaceImpl::ResumeRuntime()
     return S_OK;
 }
 
+HRESULT ProfToEEInterfaceImpl::GetEnvironmentVariable(
+    const WCHAR *szName,
+    ULONG       cchValue,
+    ULONG       *pcchValue,
+    __out_ecount_part_opt(cchValue, *pcchValue) WCHAR szValue[])
+{
+    CONTRACTL
+    {
+        // Yay!
```

Comment:
Can you please remove all the `// Yay!` comments in the code you've added? I have no idea what they are supposed to mean.

Reply:
These are copy pasted from other places. David Broman liked to leave comments like this. I believe they are about his happiness when an api didn’t have to deal with the complexities of locks, etc. 

They don’t particularly bother me but agree we shouldn’t add more.

Reply:
@janvorli Done.